src/main/resources/META-INF/elasticsearch.properties

```diff
@@ -2,7 +2,7 @@
 
 elasticsearch.host = 127.0.0.1
 elasticsearch.port = 9200
-elasticsearch.cluster.name = protocol-analyzer
+elasticsearch.cluster.name=protocol-analyzer
```

Comment:
please revert the change. All other lines have a whitespace before and after equals symbol.

Reply:
Corrected in the latest commit 013076f